pkg/entities/nfts.go

```diff
@@ -53,11 +53,20 @@ func (e *Entity) GetNFTDetail(symbol, tokenID string) (*response.IndexerNFTToken
 	// cannot find collection in db
 	if err != nil {
 		if errors.Is(err, gorm.ErrRecordNotFound) {
-			err = fmt.Errorf("database: record nft collection not found")
+			// get collection
```

Comment:
should not do like this. Error is error
Need seperate 2 case get by name and get by symbol, can create seperate function or something